### PR TITLE
Docker rebuild

### PR DESCRIPTION
**Pull request by jaskipper**
I've fixed and simplified the process of setup. Cleaned up, moved files around, etc. All Passwords are being stored in the .env file which is accessible by both the database container and the churchcrm container. The instructions in the README file won't work as is for the run command until this is merged into Master and the image is rebuilt on hub.docker.com. You can use docker-compose to build and test now. Instructions are also in the README.

### CODE REVIEW

File: build-files/Dockerfile [hunk 1]
```diff
@@ -9,8 +9,10 @@ RUN apk update && apk upgrade && \
 RUN apk add --no-cache \
             jq \
             php7-session@community \
+            php7-pdo@community \
```
Comment (crossan007) [inline build-files/Dockerfile:12]: lol @ comment on commit...

Comment (crossan007) [reply build-files/Dockerfile:12]: take a look at http://propelorm.org/documentation/

Comment (jaskipper) [reply build-files/Dockerfile:12]: Ha, yes. I've been feeling my way through it a lot. I probably need to read a LOT more documentation too. Just not enough time!

Comment (jaskipper) [reply build-files/Dockerfile:12]: Is there anything that I need to do here to merge this?

**Comment by crossan007** [comment]
There are windows style line endings in at least run_alwyas/60_churchcrm.sh

see this test for example: https://github.com/ChurchCRM/CRM/blob/master/tests/LineEndings.sh

**Review by crossan007 — APPROVED** [review]

**Pull request merged**